src/http-gateways/trustless-gateway.md

```diff
@@ -301,6 +302,32 @@ of their presence in the DAG or the value assigned to the "dups" parameter, as
 the raw data is already present in the parent block that links to the identity
 CID.
 
+## CAR `meta` (content type parameter)
+
+The `meta=eof` parameter allows clients to request the server to include additional metadata about the
+CAR to be included at the end of the response body.
+
+This parameter SHOULD only be used with CAR `version=1`.
+Values other than `eof` SHOULD be ignored.
+
+When the parameter is not set, the server must not add any extra CAR blocks to the response.
+
+The metadata block is a regular CAR block with the following properties:
+
+- CID specifies multicodec `car-metadata` (`0x04ff`), see
+  [multicodec#334](https://github.com/multiformats/multicodec/pull/334).
```

Comment:
:information_source:  flagging that this custom codec is not a hard requirement for this IPIP.

As noted in "Option 1" from https://github.com/ipfs/specs/pull/431#issuecomment-1670958141  
`meta=eof` could use regular `dag-cbor` in CID it will still be ok, because `eof` behavior  is opt-in via `Accept` request header and there is explicit signaling in `Content-Type` response header returned before CAR stream.

One could argue this is more friendly to existing implementations, as `dag-cbor` code will not create errors in alternative implementations, while a custom code might.

```suggestion
The metadata block is a regular CAR block with the following properties:

- CID specifies multicodec `dag-cbor` (`0x71`)
```